tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -14,21 +13,45 @@ import (
 	"github.com/openshift/odo/tests/helper"
 )
 
-const (
-	CI_OPERATOR_HUB_PROJECT = "ci-operator-hub-project"
-)
-
 var _ = Describe("odo service command tests for OperatorHub", func() {
 
+	var project string
+
 	BeforeEach(func() {
 		SetDefaultEventuallyTimeout(10 * time.Minute)
 		SetDefaultConsistentlyDuration(30 * time.Second)
-		helper.CmdShouldPass("odo", "project", "set", CI_OPERATOR_HUB_PROJECT)
 		// TODO: remove this when OperatorHub integration is fully baked into odo
-		os.Setenv("ODO_EXPERIMENTAL", "true")
+		helper.CmdShouldPass("odo", "preference", "set", "Experimental", "true")
```

Comment:
Preference will clash for each spec. SO you need to overwrite the preference path each time the spec is called. For example 
```
context = helper.CreateNewContext()
os.Setenv("GLOBALODOCONFIG", filepath.Join(context, "config.yaml"))
helper.CmdShouldPass("odo", "preference", "set", "Experimental", "true")
```

Reply:
Make sure you are un setting the environment variable ```GLOBALODOCONFIG``` in ```AfterEach``` teardown step too